apm-sniffer/optional-plugins/optional-spring-plugins/spring-webflux-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/spring/webflux/v5/DispatcherHandlerHandleMethodInterceptor.java

```diff
@@ -93,14 +94,11 @@ public Object afterMethod(EnhancedInstance objInst, Method method, Object[] allA
 
         AbstractSpan span = (AbstractSpan) exchange.getAttributes().get("SKYWALING_SPAN");
         
-        return ((Mono) ret).flatMap(s -> {
-                    setPattern(span, exchange);
-                    return s;
-                 })
-                .doOnError(s -> setPattern(span, exchange))
+                return ((Mono) ret)
```

Comment:
Is this the right format? I am feeling not. Could you confirm? Rather than waste CI resources.

Reply:
slightly changed code style so to doesn't looks weird now。